custom_components/ha_washdata/store_client.py

```
@@ -271,31 +320,54 @@ async def search_devices(
             rows = [r for r in rows if str(r.get("model_lc", "")).startswith(p)]
         return rows
 
-    async def list_brands(self, q: str | None = None, include_pending: bool = True, page_size: int = 60) -> list[dict[str, Any]]:
-        sq = {
-            "from": [{"collectionId": "brands"}],
-            "where": self._where([self._status_filter(include_pending)]),
-            "orderBy": [{"field": {"fieldPath": "brand_lc"}, "direction": "ASCENDING"}],
-            "limit": page_size,
-        }
-        rows = await self._run_query(sq)
+    async def list_brands(self, q: str | None = None, include_pending: bool = True, page_size: int = 500) -> list[dict[str, Any]]:
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)